resources/aws/kms.go

```diff
@@ -21,14 +27,31 @@ func (kk *KMSKey) CreateOrFail() error {
 	if err != nil {
 		return microerror.MaskAny(err)
 	}
+
+	if _, err := kk.Clients.KMS.CreateAlias(&kms.CreateAliasInput{
+		// Alias names need to start from "alias/" prefix.
+		AliasName:   aws.String(kk.fullAlias()),
+		TargetKeyId: key.KeyMetadata.Arn,
+	}); err != nil {
+		return microerror.MaskAny(err)
+	}
+
```

Comment:
I see no reason for that. I like to keep blocks like

```
something, err := Something()
if err != nil {
    return err
}
```
separated by empty lines. IMO it increases readability, it doesn't mix error handling with the other more important things.